08-communicating-results.Rmd

```diff
@@ -1,14 +1,16 @@
 # Communicating Results {#c08-communicating-results}
```

Comment:
I think this chapter has various aspects to expand upon - we can brainstorm together. Because there is (1) what to communicate - mentioned in the methodology/analysis section, and (2) how to communicate - through tables, charts, then (3) best practices - reproducibility (and there should be others?).

Some other ideas that come to mind:

- We could also talk more about interpreting statistical significance and addressing limitations and potential sources of bias.
- We could talk about storytelling in data and how to structure the narrative around key findings and insights.
- We could give an example for how to organize and structure reports or presentations (executive summary, the different headings, etc.) -> maybe can be its own vignette? We can write a real-life report based on one of the analyses?
- Any ethical considerations (communicating results responsibly and accurately, addressing biases [and how])
- We can give ideas on how to disseminate results to your audience - through webinars, interactive dashboards, etc.

Reply:
We have R Markdown/Quarto under reproducibility but it should also be a tool for disseminating results.